pkg/projects/packages.builds

```diff
@@ -2,9 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
-  <ItemGroup>
+  <PropertyGroup>
+    <BuildAllPackages>true</BuildAllPackages>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(BuildAllPackages)' == 'true'" >
     <Project Include="*\*.builds" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BuildAllPackages)' == 'false'" >
```

Comment:
This is about adding the infrastructure that we will use during servicing builds from this branch post RTM.